examples/gno.land/r/gnoland/gh/contract.gno

```diff
@@ -0,0 +1,105 @@
+package gh
+
+import (
+	"bufio"
+	"bytes"
+	"std"
+
+	"gno.land/p/demo/agent/agent"
+	ghTask "gno.land/p/demo/agent/github/verification/task"
```

Comment:
Even though this is now outdated, I partly agree with you.  The newer version of the gh verification realm defines its own `Task` implementation, so it no longer lives in the `gnorkle`.  However, I do think it would be good for us to define some basic interface implementations in this package that can be easily used by most people in most cases without having to go through the step of defining custom types.